assets/js/base/hooks/use-query-state.js

```diff
@@ -93,12 +99,15 @@ export const useQueryStateByKey = (
  * state by hydration or component attributes, or routing url changes that
  * affect query state).
  *
- * @param {string} context           What context to retrieve the query state
- *                                   for.
  * @param {Object} synchronizedQuery A provided query state object to
  *                                   synchronize internal query state with.
+ * @param {string} context           What context to retrieve the query state
```

Comment:
If context if optional, should we wrap it in brackets?
```suggestion
 * @param {string} [context]         What context to retrieve the query state
```